### ISSUE TITLE
Pitfall qemu-i386-static

### ISSUE THREAD

**Issue by pullmoll** [void-linux/void-mklive#183]
Since https://github.com/void-linux/void-mklive/commit/025d6d681ab5518c28f9be71e99fa1f9e05f2ef0 you have to run `void-mklive` alone, without at the same time trying to build packages for `i686` on an `x86_64` host. This is because `xbps-src` uses `linux32` to emulate the `i686` 32 bit environment for native builds and these two things obviously don't work together.

I fell into this pit and first thought something with the recent changes was causing the build failures for i686 packages just printing:
```
ERROR: Failed to execute command /void-packages/xbps-src (No such file or directory)
```

You need to disable `binfmt_misc` for `i686` before trying to run `./xbps-src` for linux32 again by e.g.
```
echo 0 > /proc/sys/fs/binfmt_misc/status
```

**Comment by ericonr** [comment]
See #168

**Comment by the-maldridge** [comment]
Resolved by #168.
